javaparser-testing/src/test/resources/com/github/javaparser/bdd/visitor_scenarios.story

```diff
@@ -73,4 +73,4 @@ Given a CompilationUnit
 Given a VoidVisitorAdapter with a visit method that asserts sensible line positions
 When file "JavaConcepts.java" is parsed
 When the CompilationUnit is visited by the PositionTestVisitor
-Then the total number of nodes visited is 1427
```

Comment:
This test was intended as a sort of big alarm telling how if we are dropping tons of nodes without realizing but we end up having to touch it at every change. Should we either make it more flexible (like checking the nodes are 1500 > x > 1300) or remove it?